<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ვალუტის გადამყვანი</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>691</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="login_page">
     <widget class="QPushButton" name="authorisation_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>300</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ავტორიზაცია</string>
      </property>
     </widget>
     <widget class="QLabel" name="username_input_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>400</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>მომხმარებლის სახელი:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="password_input_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>400</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>პაროლი:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_input_line">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>400</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_input_line">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>205</y>
        <width>400</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="system_message_label">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>10</y>
        <width>650</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>შეიყვანეთ მონაცემები სისტემაში შესასვლელად</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="wrong_credentials_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>400</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="currency_converter">
     <widget class="QLabel" name="currency_convert_msg_label">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>10</y>
        <width>650</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>შეიყვანეთ მონაცემები ვალუტის კონვერტაციისთვის</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="amount_input_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>შეიყვანეთ თანხა:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="amount_input_line">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="from_currency_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>საწყისი ვალუტა:</string>
      </property>
     </widget>
     <widget class="QLabel" name="to_currency_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>110</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ვალუტა კონვერტაციისთვის:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="from_currency_combobox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="to_currency_combobox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>გადაყვანა</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_button">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>310</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>გასუფთავება</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_button">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>310</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>გამოსვლა</string>
      </property>
     </widget>
     <widget class="QLabel" name="result_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>210</y>
        <width>300</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>კონვერტაციის შედეგი:</string>
      </property>
     </widget>
     <widget class="QLabel" name="result_amount">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currency_type_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>550</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>აირჩიეთ ვალუტის ტიპი, რომლის ჩვენებაც გსურთ:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="currency_type_combobox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>ყველა</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ქვეყნების</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>კრიპტო</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="defaults_button">
      <property name="geometry">
       <rect>
        <x>355</x>
        <y>310</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>გადატვირთვა</string>
      </property>
     </widget>
     <widget class="QLabel" name="cache_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>155</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>დაგროვებული ქეში:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cache_size">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>155</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="date_label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>350</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
